.travis.yml

```diff
@@ -6,9 +6,9 @@ dist: trusty
 matrix:
     include:
         # Longest build first
-        - env: PYTHON=3.5 NUMPY=1.11 RUN_COVERAGE=yes
-        - env: PYTHON=3.4 NUMPY=1.9 BUILD_DOC=yes
-        - env: PYTHON=2.7 NUMPY=1.7
+        - env: PYTHON=3.5 NUMPY=1.11 RUN_COVERAGE=yes ENV=travisci
```

Comment:
Call it CONDA_ENV or CI_ENV? ENV is very generic and might already be used as en environment variable.